mailchimp.api.php

```diff
@@ -56,3 +56,43 @@ function hook_mailchimp_subscribe_user($list_id, $email, $merge_vars) {
 function hook_mailchimp_unsubscribe_user($list_id, $email) {
 
 }
+
+/**
+ * Alter the entity options list on the automations entity form.
+ *
+ * @param array $entity_type_options
+ *   The full list of Drupal entities.
+ * @param string $automation_entity_label
+ *   The label for the automation entity, if it exists.
+ */
+function hook_mailchimp_automations_entity_options(&$entity_type_options, $automation_entity_label) {
+
+}
+
+/**
+ * Alter mergevars before a workflow automation is triggered.
+ *
+ * @param array $merge_vars
+ *   The merge vars that will be passed to MailChimp.
+ * @param object $automation_entity
+ *   The MailchimpAutomationEntity object.
+ * @param object $wrapped_entity
+ *   The EntityMetadataWrapper for the triggering entity.
+ */
+function hook_mailchimp_automations_mergevars_alter(&$merge_vars, $automation_entity, $wrapped_entity) {
+}
+
+/**
+ * Perform an action after a successful MailChimp workflow automation.
+ *
+ * @param object $automation_entity
+ *   The MailchimpAutomationEntity object.
+ * @param string $email
+ *   The email_property value from the MailchimpAutomationEntity.
+ * @param object $wrapped_entity
+ *   The EntityMetadataWrapper for the triggering entity.
+ */
+function hook_mailchimp_automations_workflow_email_triggered($automation_entity, $email, $wrapped_entity) {
+
+}
+
```

Comment:
That was my first instinct, and what I did initially. However, it appears the pattern your team has used to this point it to put api hook documentation for all the mailchimp modules in this document (@see hook_mailchimp_lists_mergevars_alter), so I moved it here. Redhen looks like it does the same. However, I checked media and they do it the other way. Seems like there's not a clear standard. Please advise.